<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QModuleRegister</class>
 <widget class="QDialog" name="QModuleRegister">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>454</width>
    <height>408</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Зарегистрировать модуль</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resources.qrc">
    <normaloff>:/images/G.png</normaloff>:/images/G.png</iconset>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>372</y>
     <width>156</width>
     <height>24</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>431</width>
     <height>201</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::WinPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Sunken</enum>
   </property>
   <property name="lineWidth">
    <number>3</number>
   </property>
   <property name="midLineWidth">
    <number>1</number>
   </property>
   <widget class="QListWidget" name="fileList">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>161</width>
      <height>161</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="fileNameLbl">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>77</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Файл модуля</string>
    </property>
   </widget>
   <widget class="QLabel" name="commentLbl">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>10</y>
      <width>78</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Комментарий</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="commentTxtEdt">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>30</y>
      <width>241</width>
      <height>161</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QFrame" name="frame_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>220</y>
     <width>431</width>
     <height>141</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::WinPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Sunken</enum>
   </property>
   <property name="lineWidth">
    <number>3</number>
   </property>
   <property name="midLineWidth">
    <number>1</number>
   </property>
   <widget class="QLabel" name="parametersLbl">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>66</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Параметры</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="parametersTable">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>411</width>
      <height>101</height>
     </rect>
    </property>
    <property name="editTriggers">
     <set>QAbstractItemView::NoEditTriggers</set>
    </property>
    <property name="showDropIndicator" stdset="0">
     <bool>false</bool>
    </property>
    <property name="dragDropOverwriteMode">
     <bool>false</bool>
    </property>
    <property name="dragDropMode">
     <enum>QAbstractItemView::NoDragDrop</enum>
    </property>
    <property name="selectionMode">
     <enum>QAbstractItemView::SingleSelection</enum>
    </property>
    <property name="selectionBehavior">
     <enum>QAbstractItemView::SelectRows</enum>
    </property>
    <attribute name="horizontalHeaderCascadingSectionResizes">
     <bool>true</bool>
    </attribute>
    <attribute name="horizontalHeaderDefaultSectionSize">
     <number>120</number>
    </attribute>
    <attribute name="horizontalHeaderHighlightSections">
     <bool>false</bool>
    </attribute>
    <attribute name="horizontalHeaderMinimumSectionSize">
     <number>40</number>
    </attribute>
    <attribute name="horizontalHeaderStretchLastSection">
     <bool>true</bool>
    </attribute>
    <attribute name="horizontalHeaderStretchLastSection">
     <bool>true</bool>
    </attribute>
    <attribute name="horizontalHeaderCascadingSectionResizes">
     <bool>true</bool>
    </attribute>
    <attribute name="horizontalHeaderMinimumSectionSize">
     <number>40</number>
    </attribute>
    <attribute name="horizontalHeaderHighlightSections">
     <bool>false</bool>
    </attribute>
    <attribute name="horizontalHeaderDefaultSectionSize">
     <number>120</number>
    </attribute>
    <column>
     <property name="text">
      <string>Параметр</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Тип</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Класс</string>
     </property>
    </column>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>QModuleRegister</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>QModuleRegister</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <designerdata>
  <property name="gridDeltaX">
   <number>10</number>
  </property>
  <property name="gridDeltaY">
   <number>10</number>
  </property>
  <property name="gridSnapX">
   <bool>true</bool>
  </property>
  <property name="gridSnapY">
   <bool>true</bool>
  </property>
  <property name="gridVisible">
   <bool>true</bool>
  </property>
 </designerdata>
</ui>
